about.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/e98a0f000a.js" crossorigin="anonymous"></script>
+    <title>Era-Project</title>
+</head>
+<body>
```

Comment:
- [ ] Please make sure to fix the overflow on the about page to match the given design, there is an over size to the right side of the page.